deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -1013,6 +1013,8 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: ocs-operator
+                - name: ROOK_CEPH_IMAGE
+                  value: rook/ceph:v1.1.2
```

Comment:
Instead of adding a new property we should be able to detect the image that is running in the operator pod. The OCS operator has privs to get deployments, right? Here's an example of where Rook looks up the image running in the operator pod: [GetOperatorImage()](https://github.com/rook/rook/blob/823018b1c8c1475fa2a1433aae3c99382c4269cf/cmd/rook/rook/rook.go#L162)

Reply:
That's another option. I'm not sure it buys us much though since this ROOK_CEPH_IMAGE value is auto populated during the CSV generation automation at the same point that we define the rook/ceph operator image that should be deployed.

Reply:
I would say it buys us something if we want to consider the notion of the ocs-operator apart from the CSV. We have no reason to do that now, I know, but it feels more logically coherent to keep them as decoupled as possible in the implementation/deployment, and leave the coordination of operators to the packaging (CSV).

Reply:
I see, as long as it's generated and doesn't have a possibility of being overlooked.